Multisig/Cross-layer/FirebaseRemoteConfig.swift

```diff
@@ -17,13 +17,16 @@ class FirebaseRemoteConfig {
         case deprecated
         case safeClaimEnabled
         case crashDebugEnabled
+        //FIXME Remove feature flag and flag handling after release
+        case intercomMigration
     }
 
     private var remoteConfig: RemoteConfig
     private let defaultValues: [String : NSObject] = [Key.newestVersion.rawValue : "" as NSObject,
                                                       Key.deprecatedSoon.rawValue : "" as NSObject,
                                                       Key.deprecated.rawValue : "" as NSObject,
-                                                      Key.safeClaimEnabled.rawValue : false as NSObject]
+                                                      Key.safeClaimEnabled.rawValue : false as NSObject,
+                                                      Key.intercomMigration.rawValue : false as NSObject] //FIXME Remove feature flag and flag handling after release
```

Comment:
the migration is done, we don't need it anymore, right?

Reply:
No, next pr